package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "leaderboard",
+  "version": "1.0.0",
+  "description": "Leaderboard",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack",
+    "start": "webpack serve --open",
+    "watch": "webapack --watch"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/mugberto/leaderboard.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/mugberto/leaderboard/issues"
+  },
+  "homepage": "https://github.com/mugberto/leaderboard#readme",
+  "dependencies": {
+    "css-loader": "^6.2.0",
+    "html-webpack-plugin": "^5.3.2",
+    "style-loader": "^3.2.1",
+    "webpack": "^5.47.0",
+    "webpack-cli": "^4.7.2"
```

Comment:
- [ ] All webpack related packages like webpack, webpack-cli, webpack-dev-server etc should be added as devDependencies. You can always use this [link](https://www.npmjs.com) `npm i` and it will always install your dependencies correctly